api-extractor/src/DocElementParser.ts

```diff
@@ -190,4 +204,87 @@ export default class DocElementParser {
 
     return linkDocElement;
   }
+
+  /**
+   * This method parses the semantic information in an \@inheritdoc JSDoc tag and sets
+   * all the relevant documenation properties from the inherited doc onto the documenation
+   * of the current api item.
+   *
+   * The format for the \@inheritdoc tag is {\@inheritdoc scopeName/packageName:exportName.memberName}.
+   * For more information on the format see IInheritdocRef.
+   */
+  public static parseInheritDoc(documentation: ApiDocumentation, token: Token): void {
+
+    // Check to make sure the API definition reference is at most one string
+    const tokenChunks: string[] = token.text.split(' ');
+    if (tokenChunks.length > 1) {
+      documentation.reportError('Too many parameters for @inheritdoc inline tag.' +
+        'The format should be {@inheritdoc scopeName/packageName:exportName}. Extra parameters are ignored');
+      return;
+    }
+
+    // Create the IApiDefinitionReference object
+    // Deconstruct the API reference expression 'scopeName/packageName:exportName.memberName'
+    const apiDefinitionRef: ApiDefinitionReference = ApiDefinitionReference.createFromString(
+      token.text,
+      documentation.reportError
+    );
+    // if API reference expression is formatted incorrectly then apiDefinitionRef will be undefined
+    if (!apiDefinitionRef) {
+      documentation.reportError('Incorrecty formatted API definition reference');
+      return;
+    }
+
+    // Atempt to locate the apiDefinitionRef
+    const resolvedApiItem: ResolvedApiItem = documentation.referenceResolver.resolve(
+      apiDefinitionRef,
+      documentation.extractor.package,
+      documentation.reportError
+    );
+
+    // If no resolvedApiItem found then nothing to inherit
+    // But for the time being set the summary to a text object
+    if (!resolvedApiItem) {
+      const textDocItem: IDocElement = {
+        kind: 'textDocElement',
+        value: `See documentation for ${tokenChunks[0]}`
+      } as ITextElement;
+      documentation.summary = [textDocItem];
+      return;
+    }
+
+    // We are going to copy the resolvedApiItem's documentation 
+    // We must make sure it's documentation can be completed,
+    // if we cannot, an error will be reported viathe documentation error handler. 
+    // This will only be the case our resolvedApiItem was created from a local 
+    // ApiItem. Resolutions from JSON will have an undefined 'apiItem' property.
+    // Example: a circular reference will report an error. 
+    if (resolvedApiItem.apiItem) {
```

Comment:
@pgonzal This is a point of interest, this is the one place (other than ApiItem) that we force  target.tryResolveReferences().